src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,211 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+
+import org.junit.jupiter.api.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private static final long MIN_ID_LENGTH = 1L;
+    private static final long MAX_ID_LENGTH = 200L;
```

Comment:
The constants `MIN_ID_LENGTH` and `MAX_ID_LENGTH` are used to represent ID values, not lengths. Consider renaming them to `MIN_ID_VALUE` and `MAX_ID_VALUE` to better reflect their purpose.